src/ui/Templates/CoreHelpers.ts

```diff
@@ -135,10 +136,10 @@ export interface ICoreHelpers {
    * Formats a numeric value using the format string.
    *
    * - `content`: The numeric value to format.
-   * - `format`: The format string to use. The options available are defined by
-   *   the [Globalize](https://github.com/klaaspieter/jquery-global#numbers) library.
+   * - `format`: Optional. The format string to use. The options available are defined by
```

Comment:
My preferred wording for the third sentence would be:
```
See the Globalize library for available options.
```

But I leave the last word to @fbeaudoincoveo,

Reply:
Suggestion:

```
- `format` Optional. The string format to use. See the <a href="https://github.com/klaaspieter/jquery-global#numbers" target="_blank">Globalize</a> library for the list of available formats.
```

Then I think I'd add a usage example of the helper, like:

> **Example:**
> ```html
> <span class="CoveoFieldValue" data-field="@viewcount" data-text-caption="Views"
>            data-helper="number" data-helper-options-format="n0"></span>
> ```

I think this would be clearer than "Either a string or an object (see INumberFormatOptions)."